pal_tiago/tiago.xml

```diff
@@ -0,0 +1,222 @@
+<mujoco model="tiago">
+
+  <compiler angle="radian" meshdir="./assets/" autolimits="true"/>
+
+  <option timestep="0.001" integrator="implicit" />
+
+  <asset>
+    <mesh name="sick_tim551" file="sensors/sick_tim551.stl"/>
```

Comment:
You can get rid of all the name attributes below. MuJoCo will automatically extract and use the filename from the path.

Reply:
Done